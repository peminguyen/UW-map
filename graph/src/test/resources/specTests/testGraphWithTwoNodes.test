CreateGraph graphWithTwoNodes

AddNode graphWithTwoNodes A
AddNode graphWithTwoNodes B

ListNodes graphWithTwoNodes
ListChildren graphWithTwoNodes A
ListChildren graphWithTwoNodes B