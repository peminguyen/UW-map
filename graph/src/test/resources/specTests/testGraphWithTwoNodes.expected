created graph graphWithTwoNodes

added node A to graphWithTwoNodes
added node B to graphWithTwoNodes

graphWithTwoNodes contains: A B
the children of A in graphWithTwoNodes are:
the children of B in graphWithTwoNodes are: